curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-use-basic-programming-vocabulary-in-conversations/6629e10635cf1ec412861da1.md

```diff
@@ -13,9 +13,9 @@ The phrase `a lot` is used to express a large quantity or degree of something. I
 
 When you want to say that there is much of something or many of something, you can use `a lot` followed by `of`, like in `a lot of books`(countable noun) or `a lot of water` (uncountable noun).
 
-You can use it at the end of the sentence to express that something is done often or frequently. An example is `He travels a lot.` It means `He travels often.`
+You can use it at the end of the sentence to express that something is done often or frequently. An example is `He travels a lot.` It means he travels often.
 
-This phrase can also mean `very much`. For example, `I like your idea a lot.` means `I like your idea very much.`
+This phrase can also mean `very much`. For example, `I like your idea a lot` means I like your idea very much.
```

Comment:
I would put these in quotes
```suggestion
This phrase can also mean `very much`. For example, `I like your idea a lot` means "I like your idea very much".
```

or

```suggestion
This phrase can also mean `very much`. For example, `I like your idea a lot` means that I like your idea very much.
```

is there somewhere a standard mentioned by the English course team?